roofit/roofitcore/test/AutomaticDiff/simple.cpp

```diff
@@ -0,0 +1,32 @@
+#include <RooRealVar.h>
```

Comment:
The convention is that the file should have the same name as the binary, ending with `cxx` not cpp. So please rename it to `AutomaticDiffSimple.cxx`.